model/Core/Vocabularies/RelationshipCompleteness.md

```diff
@@ -4,19 +4,21 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Indicates whether a relationship is complete or incomplete or if there
+can be made no assertion either way.
 
 ## Description
 
-TODO
+RelationshipCompleteness indicates whether a relationship is complete or incomplete or if there
```

Comment:
Similar comment `incomplete` -> `known to be incomplete`

Reply:
done